email_notification/tests/test_notification_builders.py

```diff
@@ -227,9 +227,7 @@ def test_get_content(self):
     def test_padding_does_not_matter_in_curly_brackets(self):
         EmailTemplate.objects.all().delete()
         content_with_padding = "I have {{ reservation_number }} padding"
-        compiled_content_with_padding = (
-            f"I have {str(self.reservation.id).zfill(10)} padding"
-        )
+        compiled_content_with_padding = f"I have {self.reservation.id} padding"
```

Comment:
Possible nitpick: What `padding` means here? If it is related to leading zeros, some variable renaming could be done here since zeroes are now gone ☺️

Reply:
yup, this is quite obfuscating name; it means that in upper:
`content_with_padding = "I have {{ reservation_number }} padding"`
has a padding in the curly brackets and the we test that it isn't affecting inside of curlys... Perhas I'll change the variable name  to `content_with_space_inside_braces` and `compiled_content_with_space_inside_braces` ? :thinking: